std/getopt.d

```diff
@@ -88,7 +88,7 @@ void main(string[] args)
  The $(D getopt) function takes a reference to the command line
  (as received by $(D main)) as its first argument, and an
  unbounded number of pairs of strings and pointers. Each string is an
- option meant to "fill" the value pointed-to by the pointer to its
+ option meant to "fill" the value pointed to by the pointer to its
```

Comment:
Ugh, pointed to by the pointer.

How about `"fill" the value referenced by the pointer to its right`